jssource/src_files/include/SubPanel/SubPanelTiles.js

```diff
@@ -283,12 +283,30 @@ function showSubPanel(child_field, url, force_load, layout_def_key) {
     }
 
     current_subpanel_url = url;
-    var returnstuff = http_fetch_sync(url + '&inline=' + inline + '&ajaxSubpanel=true');
-    request_id++;
-    got_data(returnstuff, inline);
-    if ($('#whole_subpanel_' + child_field).hasClass('useFooTable')) {
-      $('#whole_subpanel_' + child_field + ' .table-responsive').footable();
-    }
+
+    var loadingImg = '<img src="themes/SuiteP/images/loading.gif">';
```

Comment:
It would be better not to hard code the theme in the url. Consider using `SUGAR.themes.theme_name` instead of SuiteP.

Reply:
Yes that's true. Thanks.